### PR TITLE
Login Page - New Design

### PR DESCRIPTION
**Pull request by ainouzgali**

### CODE REVIEW

**Comment by scopsy** [comment]
@ainouzgali 😍 How can I view the changes? When navigating to http://localhost:4200/auth/login I'm still seeing the old design

File: apps/web/src/design-system/password-input/PasswordInput.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ */
+export const PasswordInput = React.forwardRef<HTMLInputElement, IPasswordInputProps>(
+  ({ value, onChange, ...props }: IPasswordInputProps, ref) => {
+    const defaultDesign = { radius: 'md', size: 'md', styles: inputStyles } as PasswordInputProps;
+
+    return (
+      <StyledPassword ref={ref} {...defaultDesign} onChange={onChange} autoComplete="off" value={value} {...props} />
+    );
+  }
+);
```
Comment (scopsy) [inline apps/web/src/design-system/password-input/PasswordInput.tsx:29]: Great Addition!

File: apps/web/src/components/layout/components/AuthLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+        {children}
+      </div>
+    </ThemeProvider>
+  );
+}
+
+const useStyles = createStyles((theme) => ({
+  wrapper: {
+    height: '100%',
```
Comment (scopsy) [inline apps/web/src/components/layout/components/AuthLayout.tsx:47]: CleanShot 2022-03-24 at 10 35 25
There is a strange behaviour when the height of the screen is smaller and a scroll area is generated the bg image is not stretched to the end. Maybe worth creating an overflow scroll in this case for the right form section and that it's height will also be 100% height and then overflow: auto; 

What do you think?

File: apps/web/src/components/layout/components/AuthLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          <img
+            src="/static/images/notifications/notification_02.png"
+            alt="logo"
+            style={{ position: 'absolute', left: '10%', top: '45%', maxWidth: 400 }}
+          />
+          <img
+            src="/static/images/notifications/notification_03.png"
+            alt="logo"
+            style={{ position: 'absolute', left: '7%', top: '55%', maxWidth: 400 }}
+          />
```
Comment (scopsy) [inline apps/web/src/components/layout/components/AuthLayout.tsx:36]: Small suggestion:
I would wrap the 3 images in a single div and calculate it's total height with the 3 notifications.
I would than do this div to position absolute and top: calc(100% - **heightOfBlock** / 2);

And then all the images inside will just be margin Bottom spaced without absolute. 

In this case the distance won't differ when changing screen height like in this photo:
CleanShot 2022-03-24 at 10 35 25

File: apps/web/src/components/auth/LoginForm.tsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    <>
+      <GithubButton
+        my={30}
+        variant="white"
+        fullWidth
+        radius="md"
+        leftIcon={<Github />}
+        sx={{ color: colors.B40, fontSize: '16px', fontWeight: '700', height: '50px' }}>
+        Sign In with Github
+      </GithubButton>
```
Comment (scopsy) [inline apps/web/src/components/auth/LoginForm.tsx:61]: Is this possible to wrap it, or make it as "a" take and do a link to "API_URL/v1/auth/github"?

**Review by scopsy — COMMENTED** [review]
This looks amazing, love the new design so much! 

There is a small issue when the height of the screen changes a bit, added some comments regarding that.

File: apps/web/src/components/layout/components/AuthLayout.tsx (hunk 2 above)
Comment (ainouzgali) [reply apps/web/src/components/layout/components/AuthLayout.tsx:47]: Sounds good!

File: apps/web/src/components/auth/LoginForm.tsx (hunk 4 above)
Comment (ainouzgali) [reply apps/web/src/components/auth/LoginForm.tsx:61]: yes :)

**Comment by scopsy** [comment]
@ainouzgali Looks amazing! A small and strange issue i'm experiencing is that none of the text is selectable in the page.
I think that this happens because there DIV with the background image is on top of all of the content and prevents from selecting text and clicking some of the buttons
CleanShot 2022-03-24 at 16 26 27
.

**Review by scopsy — APPROVED** [review]
Looking forward for it!!

**Pull request merged**